README.md

```diff
@@ -32,10 +35,10 @@ registration.agree_terms
 # We need to prove that we control the domain using one of the challenges method.
 authorization = client.authorize(domain: 'example.org')
 
-# For now the only challenge method supprted by the client is http-01.
+# For now the only challenge method supported by this client is http-01.
```

Comment:
https://github.com/unixcharles/acme-client/blob/master/lib/acme/client/resources/authorization.rb#L19-L21

I'm not sure I'd add more examples here, it's probably time for some more structured documentation than a single big example.